### PR TITLE
fix: Ctrl-C does not interrupt create prompt

### LINKED ISSUE #1887: Hitting <Ctrl>+C on the `limactl start` interactive prompt does not cancel the operation
### Steps to reproduce:

1. Create a new lima VM with `limactl start --name=nameofVMthatdoesntexistyet`
2. Wait for prompt to appear and then press `<Ctrl>+C`

### Expected result:

- The user is returned back to the shell prompt and the lima VM creation process does NOT initiate

### Actual result:

- The lima VM creation process continues as if `<Enter>` was pressed, accepting the `Proceed with current configuration` prompt

Details:
- OS: Archlinux
- lima version: 0.17.2

### PR DESCRIPTION
**Pull request by cwegener**
Fixes #1887

### CODE REVIEW

File: cmd/limactl/start.go [hunk 1]
```diff
@@ -376,6 +377,9 @@ func chooseNextCreatorState(st *creatorState, yq string) (*creatorState, error)
 			},
 		}
 		if err := survey.AskOne(prompt, &ans); err != nil {
+			if err == terminal.InterruptErr {
+				logrus.Fatal("interrupted")
```
Comment (AkihiroSuda) [inline cmd/limactl/start.go:381]: Maybe
```suggestion
				logrus.Fatal("terminal was interrupted")
```

Comment (cwegener) [reply cmd/limactl/start.go:381]: I think that the "more correct" message probably would be:

`Interrupted by user`

WDYT?

Comment (AkihiroSuda) [reply cmd/limactl/start.go:381]: SGTM

**Review by AkihiroSuda — APPROVED** [review]
Thanks

**Pull request merged**